src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/DragAndDropTests/DragDrop_ListViewReorder_Automated.cs

```diff
@@ -77,6 +78,7 @@ public void Test_Reorder(int from, int to)
 			var result = TakeScreenshot("Result", ignoreInSnapshotCompare: true);
 
 			ImageAssert.HasColorAt(result, x, dstY, _items[from], tolerance: 10);
+			Assert.AreEqual("Operation", op.GetDependencyPropertyValue<string>("Text"));
```

Comment:
Should the text in fact be "Move" here?

Some of the tests are failing because the actual value is "Move", but some are failing because it's "None" - is the latter a real bug? 

https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=33932&view=ms.vss-test-web.build-test-results-tab&runId=1603314&paneView=debug&resultId=100117

Reply:
Hummmm definitely !